tests/test-cases/Group1-Docker-Commands/1-10-Docker-PS.robot

```diff
@@ -108,6 +108,9 @@ Docker ps powerOn container OOB
 
     Wait Until Keyword Succeeds  10x  6s  Assert Number Of Containers  ${len+1}
 
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} stop jojo
```

Comment:
Why is this step needed?

Reply:
There is a bug with docker kill and OOB stopped containers.